src/partials/results.html

```diff
@@ -1,11 +1,15 @@
-<section class="lists">
+<section class="lists" ng-controller="searchResults()">
   <h2>Results</h2>
-  <ul>
-    <li>
+  <ul class="search-menu">
+    <li ng-repeat="result in results">
       <a href="#">
-        <img src="{{USE.API.IMG}}" />
-        <h3>{{RESULTS.API.TITLE}}</h3>
-        <p>{{RESULTS.API.CURRENT_PRICE}}</p>
+        <img ng-src="{{product.image_thumbnail}}"></img>
+        <div class="product-txt">
+          <h3>{{product.title}}</h3>
+          <p id="price">{{product.current_price | currency}}</p>
```

Comment:
Just leaving this here... :trollface: 

http://stackoverflow.com/questions/9632311/which-html5-tags-are-more-appropriate-for-money